requirements.txt

```diff
@@ -12,3 +12,4 @@ django-private-storage~=2.2
 celery~=4.4.0
 redis~=3.5.0
 django-ipware~=3.0.0
+drf-yasg~=1.17
```

Comment:
this requirement is defined in openwisp-utils[rest]: https://github.com/openwisp/openwisp-utils/blob/master/setup.py#L58